sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -257,9 +257,10 @@ class ResolveSessionCatalog(
         case v1Table: V1Table =>
           DescribeColumnCommand(tbl.asTableIdentifier, colNameParts, isExtended)
       }.getOrElse {
-        if (isTempView(tbl)) {
```

Comment:
can we follow how we deal with `CacheTableStatement`? Basically just call `parseTempViewOrV1Table`